dynawo/sources/Solvers/VariableTimeStep/SolverIDA/DYNSolverIDA.cpp

```diff
@@ -463,9 +574,13 @@ SolverIDA::calculateIC(double tEnd) {
 
   // reinit output
   flagInit_ = false;
-#if _DEBUG_
-  solverKINNormal_->setCheckJacobian(false);
-#endif
+  if (activateCheckJacobianAfterInit_) {
+    solverKINNormal_->setCheckJacobian(true);
```

Comment:
I would propose solverKINNormal_->setCheckJacobian(activateCheckJacobianAfterInit_) and same for the other one (same for line 480)